cmd/brew-bundle.rb

```diff
@@ -71,6 +73,8 @@
     Bundle::Commands::Cleanup.run
   when "check"
     Bundle::Commands::Check.run
+  when "wrap"
+    Bundle::Commands::Wrap.run ARGV[1..-1]
```

Comment:
Added with a little tweaking, to make sure it works if an option precedes the command name.